src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -442,6 +458,52 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
         }
     }
 
+    /// <summary>
+    /// WorkspaceWriteConverter is used for serializing Workspaces to JSON.
+    /// </summary>
+    public class DummyNodeWriteConverter : JsonConverter
+    {
+        //private EngineController engine;
+
+        public DummyNodeWriteConverter(EngineController engine = null)
+        {
+            //if (engine != null)
```

Comment:
engine is required here?

Reply:
This was a vestigial remnant from copying another converter. Engine is not needed here, or the default constructor, I have removed both.